api/tests/organizational-entities/unit/domain/models/OrganizationForAdmin_test.js

```diff
@@ -667,10 +667,23 @@ describe('Unit | Organizational Entities | Domain | Model | OrganizationForAdmin
         provinceCode: initialProvinceCode,
       });
       // when
-      givenOrganization.updateProvinceCode(newProvinceCode);
+      givenOrganization.provinceCode = newProvinceCode;
       // then
       expect(givenOrganization.provinceCode).to.equal(newProvinceCode);
     });
+
+    it('fix provinceCode format when there is no minimum 3 numbers', function () {
+      // given
+      const initialProvinceCode = '6';
+      const newProvinceCode = '44';
+      const givenOrganization = new OrganizationForAdmin({
+        provinceCode: initialProvinceCode,
+      });
+      // when
+      givenOrganization.provinceCode = newProvinceCode;
+      // then
+      expect(givenOrganization.provinceCode).to.equal(`0${newProvinceCode}`);
```

Comment:
Le test ici serait peut-être un peu plus lisible en mettant explicitement la valeur attendue qui est `044` :

```suggestion
      expect(givenOrganization.provinceCode).to.equal(`044`);
```